firecrown/modeling_tools.py

```diff
@@ -29,13 +29,17 @@ def __init__(
         self,
         *,
         pt_calculator: None | pyccl.nl_pt.EulerianPTCalculator = None,
+        hm_calculator: None | pyccl.halos.HMCalculator = None,
+        cM_relation: None | str = None,
```

Comment:
We would prefer for the cM_relation to be the concrete `Concentration` object, rather than a string which will eventually be converted to the right object.

Reply:
It can also be a `Concentration` object but then it has to be defined at the likelihood building level and I am not sure how it would be moved under the `WeakLensingTracer` object in this case. But I can imagine it's possible.